node_modules/oae-core/topnavigation/topnavigation.html

```diff
@@ -5,34 +5,14 @@
     <!-- ACCESSIBILITY HELPERS -->
     <a id="topnavigation-skip" class="sr-only oae-skip-link hide" href="#topnavigation-target" accesskey="__MSG__SKIP_TOP_MENU_KEY__">__MSG__SKIP_TOP_MENU__</a>
     <!-- NAVIGATION -->
-    <div class="row">
-        <!-- LEFT MENU -->
-        <div id="topnavigation-left" class="col-xs-5"><!-- --></div>
-        <!-- LOGO -->
-        <div id="topnavigation-institutional-logo-container" class="col-xs-2 text-center"><!-- --></div>
-        <!-- RIGHT MENU -->
-        <div id="topnavigation-right" class="col-xs-5 text-right">
-            <!-- SEARCH FORM -->
-            <form action="." id="topnavigation-search-form" class="form-inline" role="search">
-                <div class="form-group">
-                    <input type="text" id="topnavigation-search-query" name="topnavigation-search-query" class="search-query form-control pull-left" title="__MSG__SEARCH_FOR_CONTENT_FOLDERS_PEOPLE_GROUPS__"/>
-                    <button type="submit" id="topnavigation-search-icon" class="btn btn-link pull-left" title="__MSG__SEARCH__">
-                        <i class="fa fa-search"><span class="sr-only">__MSG__SEARCH__</span></i>
-                    </button>
-                </div>
-            </form>
-        </div>
-    </div>
+    <div class="row"><!-- --></div>
 </div>
 <h1 id="topnavigation-target" class="sr-only">__MSG__SKIP_MENU_TARGET__</h1>
 
 <div id="topnavigation-institutional-logo-template"><!--
-    <a href="/" class="oae-institutional-logo hidden-xs visible-sm" title="${oae.data.me.tenant.displayName|encodeForHTMLAttribute}">
-        <span class="sr-only">${oae.data.me.tenant.displayName|encodeForHTML}</span>
-    </a>
-    <a href="/" class="oae-institutional-logo-small visible-xs" title="${oae.data.me.tenant.displayName|encodeForHTMLAttribute}">
+    <span class="oae-institutional-logo hidden-xs visible-sm" title="${oae.data.me.tenant.displayName|encodeForHTMLAttribute}">
```

Comment:
In hindsight, the institutional logo has to remain a link to `/` to ensure that anonymous users can get back to the tenant landing page. In that case, we can make also make the institutional logo a link to `/` for authenticated users for consistency sake.